felipeknorrkuhn/FELIPE_KNORR_KUHN.md

```diff
@@ -0,0 +1,19 @@
+# Automation against humanity
+
+Stress. Anxiety. Anger. Frustration. None of these words should be associated with a shopping experience. Still, when the COVID-19 pandemic started, several supply chains were affected, causing high demand and low stock for electronics, food, and general household goods worldwide. From toilet paper to flour to video games and N95 masks - at traditional brick and mortar stores as well as e-commerce, it was a real challenge to stock up for the months full of unknowns ahead of us.
+
+Talking about videogames more specifically, during the first weeks of March 2020, the demand for a Nintendo Switch fitness game called Ring Fit Adventure was so high in countries already under shelter-in-place orders that it went out of stock globally. Getting out of the door to exercise was not going to be possible until further notice, and people who wanted a copy of the game were out of luck unless they paid two or three times over retail price for one. It took several months for additional units to become available, and only over one year after the pandemic started, some stores marked down the game's price as the interest - and the sense of necessity - faded away.
+
+Fast forward a few months, in November 2020, the Sony PlayStation 5 and the Microsoft Xbox One Series consoles launched, but getting ahold of them was, again, almost as hard as finding water in the desert. Gamers, who were still under the shelter-in-place orders, were desperate to buy their units, but something interesting and unfortunate prevented them from ordering them.
```

Comment:
ahold should be a hold

Reply:
Today I learned that `ahold` is Americanism - I'll change it :)

https://www.grammarly.com/blog/ahold-vs-a-hold/#:~:text=A%20hold%20means%20a%20grasp,widely%20used%20outside%20the%20U.S.